shared/src/packages/articles/libs/validation-schemas/article-create.validation-schema.ts

```diff
@@ -28,7 +28,7 @@ const articleCreate = joi.object<ArticleRequestDto, true>({
       'string.empty': ArticleValidationMessage.ARTICLE_TEXT_REQUIRE,
     }),
   promptId: joi.number().integer().positive().required().allow(null),
-  genreId: joi.number().integer().positive().required().messages({
+  genreId: joi.number().integer().positive().messages({
```

Comment:
I believe `article genre id required` message on empty string is not necessary now + add `.allow(null)`